src/lib/style/components/_user-search-box.scss

```diff
@@ -21,3 +21,9 @@
     }
   }
 }
+
+.UserSearchBoxV2 {
+  .SRC-userCard {
+    color: unset !important;
+  }
+}
```

Comment:
Fixes styling on card, which has high specificity from our app-wide link styling

Reply:
Is it styled properly on hover?

Reply:
Need to add `text-decoration-color: unset !important;` but otherwise yes